client/src/types/select-options-item.model.ts

```diff
@@ -4,6 +4,7 @@ export interface SelectOptionsItem {
   description?: string
   group?: string
   groupLabel?: string
+  groupOrder?: number
```

Comment:
Don't add this to the global type, it's only supported on `select-languages` component

Reply:
Without this, I cannot compile the code due to an error in : `this.availableLanguages = [ { label: $localize`Unknown language`, id: '_unknown', group: this.allLanguagesGroup, groupOrder: this.allLanguagesGroupOrder } ]` (namely the fact there is no `groupOrder` attribute in the `SelectOptionsItem`.

Reply:
Because you need to extend `availableLanguages` type: `availableLanguages: (SelectOptionsItem & { groupOrder: number })[]`